### PR TITLE
Better actionable error message when druid services are not running

### PR DESCRIPTION
**Pull request by abhishekagarwal87**
We have seen that the first-time users often don't know the next steps if druid services are unresponsive for some reason. This PR makes some of those messages a bit more clear. 

This PR has:

- [x] been self-reviewed.

### CODE REVIEW

**Review by vogievetsky — APPROVED** [review]

File: services/src/main/java/org/apache/druid/server/router/QueryHostFinder.java [hunk 1]
```diff
@@ -99,14 +99,23 @@ public Server findServerSql(SqlQuery sqlQuery)
   public <T> Server pickServer(Query<T> query)
   {
     Server server = findServer(query);
-    assertServerFound(server, "No server found for query[%s]", query);
+    assertServerFound(
+        server,
+        "There are no brokers to connect to for query[%s]."
+        + "Please check that the brokers are "
```
[the remark this answers is not in the document]
Comment (vogievetsky) [reply services/src/main/java/org/apache/druid/server/router/QueryHostFinder.java:105]: it should be `your` right?

[the remark this answers is not in the document]
Comment (techdocsmith) [reply services/src/main/java/org/apache/druid/server/router/QueryHostFinder.java:105]: Yes, thank you. modified the suggestion.

**Comment by kfaraz** [comment]
+1 for suggestions from @techdocsmith .

**Review by kfaraz — APPROVED** [review]
LGTM

**Comment by abhishekagarwal87** [comment]
Ignoring the CC failure.

**Pull request merged**